packages/select/test/validation.common.js

```diff
@@ -1,84 +1,29 @@
 import { expect } from '@esm-bundle/chai';
-import {
-  enterKeyDown,
-  escKeyDown,
-  fixtureSync,
-  nextRender,
-  nextUpdate,
-  oneEvent,
-  outsideClick,
-} from '@vaadin/testing-helpers';
+import { fixtureSync, nextRender, nextUpdate, outsideClick } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import sinon from 'sinon';
 import '@vaadin/item/vaadin-item.js';
 import '@vaadin/list-box/vaadin-list-box.js';
 
 describe('validation', () => {
-  let select, valueButton, validateSpy, changeSpy;
+  let select, validateSpy, changeSpy;
 
   describe('basic', () => {
     beforeEach(async () => {
       select = fixtureSync('<vaadin-select></vaadin-select>');
       select.items = [
-        { label: 'Option 1', value: '' },
+        { label: 'Option 1', value: 'option-1' },
         { label: 'Option 2', value: 'option-2' },
-        { label: 'Option 3', value: 'option-3' },
       ];
       await nextRender();
-      valueButton = select.querySelector('vaadin-select-value-button');
       validateSpy = sinon.spy(select, 'validate');
       changeSpy = sinon.spy();
       select.addEventListener('change', changeSpy);
     });
 
-    it('should pass the validation when the field is valid', () => {
-      select.validate();
-      expect(select.checkValidity()).to.be.true;
-      expect(select.invalid).to.be.false;
-    });
-
-    it('should not pass the validation when the field is required and has no value', async () => {
-      select.required = true;
-      await nextUpdate(select);
-
-      enterKeyDown(valueButton);
-      await oneEvent(select._overlayElement, 'vaadin-overlay-open');
-
-      escKeyDown(select._items[2]);
```

Comment:
Note, this test implicitly asserted that validation happens on close with <kbd>Escape</kbd>. We already have a test that asserts that validation happens on close with outside click. Given that `select` doesn't have any specific Escape logic, I considered it's enough to only have a test for outside click.